content_api/items/resource.py

```diff
@@ -36,7 +36,7 @@
     'priority': {'type': 'integer'},
     'uri': {'type': 'string'},
     'usageterms': {'type': 'string'},
-    'version': {'type': 'integer', 'required': True},
+    'version': {'type': 'string', 'required': True},
```

Comment:
We should add `'empty': False and 'nullable': False` as per Cerberus validation specification for string if we want to allow valid string.